docs/index.rst

```diff
@@ -88,36 +148,77 @@ The plugin can also play the file in a loop which means it can start again if en
                 It is left to the user to ensure there are no missing values in CSV file. However if the option selected is report. Then plugin will check for NaN's and report error to user. This can serve as a way to check the CSV file for missing values. However the user has to take action on what to do with NaN values. The default action is to ignore them.
                 When error is reported the user must delete the south service and try again with clean CSV file.
 
-  - **'repeatLoop': type: boolean default: false**:
-                Read CSV in a loop i.e. on reaching End Of File, again go back to beginning of the file.
+   - **'postProcessMethod': type: enumeration default: 'continue_playing'**:
+                It is the method to process the CSV file once all rows are ingested.
+                It could be:
+
+                1. continue_playing
+
+                   Play the file again if finished.
+
+                2. delete
 
+                   Delete the played file once finished.
+
+                3. rename
+
+                   Rename the file with suffix after playing.
+
+
+   - **'suffixName': type: string default: '.tmp'**:
+                The suffix name for renaming the file if postProcess method is rename.
 
 Execution
 ---------
 
-Assuming you have a csv file named vibration.csv inside FLEDGE_ROOT/data. Use the following command to use the plugin.
+Assuming you have a csv file named vibration.csv inside FLEDGE_ROOT/data/csv_data. The csv file has fixed number of columns per row.  Also assuming the column names are present in the first line. The plugin will rename the file with suffix .tmp after playing. Here is the curl command for that.
 
     .. code-block:: console
 
-        curl -sX POST http://localhost:8081/fledge/service -d '{"name":"My_south","type":"south","plugin":"csvplayback","enabled":false,"config":{"assetName":{"value":"My_csv_asset"},  "csvFilename":{"value":"vibration.csv"}, "repeatLoop":{"value":"true"}, "ingestMode":{"value":"burst"}}}' |jq
-
+       res=$(curl -sX POST http://localhost:8081/fledge/service -d  @- << EOF | jq '.'
+       {
+        "name":"My_south",
+        "type":"south",
+        "plugin":"csvplayback",
+        "enabled":false,
+        "config": {
+             "assetName":{"value":"My_csv_asset"},
+             "csvDirName":{"value":"FLEDGE_DATA/csv_data"},
+             "csvFileName":{"value":"vib"},
```

Comment:
csv filename is "vibration.csv" as per line 174.

Reply:
explanation given .

Reply:
This is a strange and confusing feature of searching files with given prefix when field name is "csvfilename". Not sure why such prefix based search is needed? And if multiple match, pick first match, this is very confusing behavior. Does user really want multiple files to be processed? If so, call the parameter to indicate the same, filename pattern or something like that.